src/main/scala/mesosphere/marathon/core/volume/providers/Helpers.scala

```diff
@@ -23,7 +23,7 @@ protected[volume] abstract class AbstractExternalVolumeProvider(
 protected[providers] object OptionSupport {
   import OptionLabelPatterns._
 
-  def validIfDefined[T](implicit validator: Validator[T]): Validator[Option[T]] = new Validator[Option[T]] {
+  def ifDefined[T](implicit validator: Validator[T]): Validator[Option[T]] = new Validator[Option[T]] {
```

Comment:
This is the same as https://github.com/mesosphere/marathon/blob/master/src/main/scala/mesosphere/marathon/api/v2/Validation.scala#L21